src/note.ts

```diff
@@ -600,6 +601,32 @@ export abstract class Note extends Tickable {
     this.preFormatted = value;
   }
 
+  // Get the number of beams for this duration
+  getBeamCount(): number {
+    const glyph = this.getGlyph();
+
+    if (glyph) {
+      return glyph.beam_count;
+    } else {
+      return 0;
+    }
+  }
+
+  // Get and set the note's `Stem`
+  getStem(): Stem | undefined {
+    return undefined;
+  }
+
+  // Get StemX
+  getStemX(): number {
+    throw new Vex.RERR('NoStem', 'No stem attached to this note.');
+  }
+
+  // Set the direction of the stem
+  setStemDirection(direction: number): this {
+    throw new Vex.RERR('NoStem', `Calling getStemDirection(${direction}) in Note.`);
+  }
```

Comment:
Can we remove all these calls? I don't think we need them anymore.

Reply:
These calls will be used if we assign by error a `Note` to `Beam` (`Note` included in `Voice`) producing a `RERR`. Otherwise it seems that we will get a TypeError specific error at run-time. The jsfiddle below show how `instanceof` and functions get called when using `as`:
 
https://jsfiddle.net/o8yav0er/

If you want to go the TypeError specific error and not RERR it seems that we can delete them.

Reply:
Thanks for the explanation. `TypeError` is definitely better -- let's remove this and have TS deal with it. :-)

Reply:
Changes on `note` reverted to master.